composer.json

```diff
@@ -18,7 +18,8 @@
         }
     ],
     "require": {
-        "semsol/arc2": "v2.2.4"
+        "semsol/arc2": "v2.2.4",
+        "chrisboulton/php-resque": "dev-master#98fde571db008a8b48e73022599d1d1c07d4a7b5"
```

Comment:
Pinned to a commit rather than 1.2?  I'm not against it, just wondering the rationale in this instance - is there a particular feature/bugfix needed?  If it's primarily to keep things up to date then there are more recent changes to php resque which we could pin to.